### ISSUE TITLE
python: can't open file 'train.py': [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by NileshPranami** [magic282/NeuSum#10]
SAVEPATH=/workspace/data/cnndm/models/neusum
+ mkdir -p /workspace/data/cnndm/models/neusum
+ python train.py -save_path /workspace/data/cnndm/models/neusum -online_process_data -max_doc_len 80 -train_oracle /workspace/data/cnndm/train/train.rouge_bigram_F1.oracle.F1mmrTrue.regGain -train_src /workspace/data/cnndm/train/train.txt.src -train_src_rouge /workspace/data/cnndm/train/train.rouge_bigram_F1.oracle.F1mmrTrue.regGain -src_vocab /workspace/data/cnndm/train/vocab.txt.100k -train_tgt /workspace/data/cnndm/train/train.txt.tgt -tgt_vocab /workspace/data/cnndm/train/vocab.txt.100k -layers 1 -word_vec_size 50 -sent_enc_size 256 -doc_enc_size 256 -dec_rnn_size 256 -sent_brnn -doc_brnn -dec_init simple -att_vec_size 256 -norm_lambda 20 -sent_dropout 0.3 -doc_dropout 0.2 -dec_dropout 0 -batch_size 64 -beam_size 1 -epochs 100 -optim adam -learning_rate 0.001 -halve_lr_bad_count 100000 -gpus 0 -curriculum 0 -extra_shuffle -start_eval_batch 1000 -eval_per_batch 1000 -log_interval 100 -log_home /workspace/data/cnndm/models/neusum -seed 12345 -cuda_seed 12345 -pre_word_vecs_enc /workspace/data/cnndm/glove/glove.6B.50d.txt -freeze_word_vecs_enc -dev_input_src /workspace/data/cnndm/dev/val.txt.src.shuffle.4k -dev_ref /workspace/data/cnndm/dev/val.txt.tgt.shuffle.4k -max_decode_step 3 -force_max_len
python: can't open file 'train.py': [Errno 2] No such file or directory

**Comment by NileshPranami** [comment]
Resolved
